galley/pkg/config/source/kube/fs/source_test.go

```diff
@@ -70,7 +68,6 @@ func TestInitialFile(t *testing.T) {
 	acc.Clear()
 
 	deleteFiles(t, dir, "foo.yaml")
-	appsignals.Notify("test", syscall.SIGUSR1)
```

Comment:
Please do not remove these. Instead add a new test (or tests) that specifically tests the watcher specific functionality.